### ISSUE TITLE
Moving repository out

### ISSUE THREAD

**Issue by laumann** [laumann/compiletest-rs#236]
This is sort of a call for maintainers, but given that there already active maintainers I'm not sure what box to put this in :sweat_smile: 

I've been thinking it would make sense to move this repository into another organization/project, as I'm honestly not actively doing anything to maintain it, and I don't see myself getting involved in maintaining it again.

I could either transfer ownership of the repo to a single user, or someone could create an organization and I could transfer the repo there. (I could, of course, also transfer it to an existing organization.)

I'm tagging some people that are either actively maintaining this project, or that I know have been involved at some point. Input is welcome.

@Munksgaard 
@Manishearth 
@phansch 
@messense 
@RalfJung 
@SergioBenitez

**Comment by Munksgaard** [comment]
An "easy" solution is to hear if any of the active contributors (like @phansch) would want merge/publish access. Then it wouldn't have to be moved from your account @laumann, at least not to begin with.

**Comment by Manishearth** [comment]
Yeah i think we can keep it here for now and expand the maintainer base.

It might be worth moving this into rust-lang given that clippy relies on it so much, and eventually we can try to move rust-lang's compiletest out of tree into this repo.

**Comment by laumann** [comment]
I really want to move it out from my account, can't we just do that? @Munksgaard or @Manishearth I'd be happy to transfer the repo to one of you, I think it should preserve issues and PRs.

**Comment by Manishearth** [comment]
That's fine by me

**Comment by Munksgaard** [comment]
Same with me. You're more active than me, @Manishearth, should we move it to your account?

**Comment by Manishearth** [comment]
Sure. We can then consider moving it to rust-lang

**Comment by Manishearth** [comment]
I need to be an admin to be able to move it

**Comment by laumann** [comment]
I'll move it, don't worry.

**Comment by laumann** [comment]
@Manishearth All right, I started the transfer to you, I think you need to accept it.

**Comment by laumann** [comment]
I've also disabled the Travis CI and AppVeyor builds.

**Comment by Munksgaard** [comment]
Huh, that was easy :smile:

**Comment by Manishearth** [comment]
cc @rust-lang/clippy do y'all think we should move this into rust-lang proper? And maybe eventually work on moving rustc's implementation out.

**Comment by Manishearth** [comment]
@rust-lang/rust-clippy
